### ISSUE TITLE
Add fs-backup-timeout to podTemplateConfig

### ISSUE THREAD

**Issue by kaovilai** [vmware-tanzu/velero#6700]
**Describe the problem/challenge you have**

Add fs-backup-timeout to podTemplateConfig to make configuring easier.

See: https://github.com/openshift/oadp-operator/pull/1102/files#diff-992e40b76fc7cf12af63ec6c2cdcc70321669228469781c0d2bc527dc4c28627R440

**Vote on this issue!**

This is an invitation to the Velero community to vote on issues, you can see the project's [top voted issues listed here](https://github.com/vmware-tanzu/velero/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).  
Use the "reaction smiley face" up to the right of this comment to vote.

- :+1: for "The project would be better with this feature added"
- :-1: for "This feature will not enhance the project in a meaningful way"

**Comment by kaovilai** [comment]
unstale

**Comment by kaovilai** [comment]
unstale

**Comment by reasonerjt** [comment]
@kaovilai 
Could you rephrase the issue to make it more reasonable to velero users?
I assume you wanna expose the parameter to `velero install` ?

**Comment by kaovilai** [comment]
No, this is just for users importing velero install package in go.

**Comment by kaovilai** [comment]
Will update description

**Comment by kaovilai** [comment]
Not an issue: There's already a podTemplateConfig func [here](https://github.com/kaovilai/velero/blob/e06cc8c76886b83ca9e0432d3cd461a1093ee6eb/pkg/install/deployment.go#L128-L132)
